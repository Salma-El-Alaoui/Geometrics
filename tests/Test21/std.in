LOAD test19.txt
LIST
MOVE c1 1 1
MOVE r1 1 1
MOVE l1 1 1
MOVE pl1 1 1
MOVE oa1 1 1
LIST
UNDO
UNDO
UNDO
UNDO
UNDO
LIST
EXIT
