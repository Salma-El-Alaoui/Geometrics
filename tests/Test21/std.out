OK
#Fichier charge
C c1 1 1 1
L l1 1 1 1 1
OA oa1 c1 l1 r1
PL pl1 1 1 1 1
R r1 1 1 1 1
OK
#c1 deplace
OK
#r1 deplace
OK
#l1 deplace
OK
#pl1 deplace
OK
#oa1 deplace
C c1 3 3 1
L l1 3 3 3 3
OA oa1 c1 l1 r1
PL pl1 2 2 2 2
R r1 3 3 3 3
OK
#Undone
OK
#Undone
OK
#Undone
OK
#Undone
OK
#Undone
C c1 1 1 1
L l1 1 1 1 1
OA oa1 c1 l1 r1
PL pl1 1 1 1 1
R r1 1 1 1 1
